app/uk/gov/hmrc/merchandiseinbaggage/model/api/Declaration.scala

```diff
@@ -36,7 +36,8 @@ case class Declaration(
   maybeTotalCalculationResult: Option[TotalCalculationResult] = None,
   emailsSent: Boolean = false,
   paymentSuccess: Option[Boolean] = None,
-  lang: String = "en")
+  lang: String = "en",
+  source: Option[String] = None)
```

Comment:
I don't think it should be Optional, or is this just for backwards compatibility?

Reply:
backwards compatibility